### ISSUE TITLE
Delete a specif value from flexdatalist

### ISSUE THREAD

**Issue by SureshHV** [sergiodlopes/jquery-flexdatalist#139]
Hi,

Iam using flexdatalist mulltiple and I want to delete a specific value when a button is clicked. 

I am able to delete and value by clicking on the X button which is placed after each value but I need this to happen when Iam clicking on the button I should be able to delete a specific value. Could you provide me some help on this?

**Comment by SureshHV** [comment]
Can anybody help me to solve this?

**Comment by SureshHV** [comment]
To delete a specific value I am using $('#keyacc').val('remove' ,'10045'); but this clears all the selected values. Can any one help me to delete a specific values from the multiple selection? This has been pending for a long time. Any help here would be highly appreciated

**Comment by sergiodlopes** [comment]
Hi,

Still having this problem, with a latest release?

**Comment by SureshHV** [comment]
Yes sir, I'm not able delete a specific value. Remove property is not
working.

**Comment by biswa34** [comment]
Hi, I have got the same issue.

I am using flexdatalist multiple and While I am trying to delete single value from the array of values in datalist all the values are removed. Kindly help to resolve this. @sergiodlopes

**Comment by LeonardoFeriato** [comment]
Hi, did you solve it?
